.github/workflows/check.yml

```diff
@@ -31,9 +29,8 @@ jobs:
             os_name: Linux
           - os: macos-latest
             os_name: macOS
-          # UPSTREAM: See above comment.
-          # - os: windows-latest
-          #   os_name: Windows
+          - os: windows-latest
+            os_name: Windows
```

Comment:
Fwiw id prefer not to run windows/mac stuff for cost reasons (they are around 10x linux machines iirc)

Reply:
I'll disable here and on the other repo